src/main/java/org/mskcc/cbio/oncokb/domain/TokenStats.java

```diff
@@ -141,4 +143,24 @@ public String toString() {
             ", usageCount=" + getUsageCount() +
             "}";
     }
+
+    public String toTabDelimited() {
+        return String.format("%1$-50s\t%2$-15s\t%3$-20s\t%4$-10s\t%5$-36s\t%6$s",
```

Comment:
Just curious why you designed this way? Why not just wrap using `\t`?

Reply:
[2022-01-27 test.txt](https://github.com/oncokb/oncokb-public/files/7972816/2022-01-27.test.txt)

Here's a test file I sent to my personal bucket. It just looks a bit cleaner and easy to read with the columns. I guess most of this data won't be read manually but it may be from time to time?